static/app/components/organizations/pageFilters/parse.tsx

```diff
@@ -117,35 +120,101 @@ function getUtcValue(maybe: boolean | ParamValue) {
   return parseUtcValue(result);
 }
 
-type ParsedParams = {
-  start?: string;
-  end?: string;
-  period?: string;
-  utc?: string;
-  [others: string]: string | null | undefined;
-};
+/**
+ * Normalizes a string or string[] into the project list parameter
+ */
+function getProject(maybe: ParamValue) {
```

Comment:
should we name it value? 🤷‍♀️

Reply:
I was keeping it consistent with everything else in this file